dashboard/app/views/layouts/application.html.haml

```diff
@@ -46,6 +46,7 @@
     = render inline: File.read(Rails.root.join('..', 'shared', 'haml', 'answerdash.haml')), type: :haml if view_options[:answerdash]
     = javascript_include_tag 'application', 'data-ot-ignore' => ''
     %script{src: webpack_asset_path("js/#{js_locale}/common_locale.js"), 'data-ot-ignore' => ''}
+    %script{src: webpack_asset_path("js/#{js_locale}/signup_locale.js"), 'data-ot-ignore' => ''}
```

Comment:
oh shoot, I missed this in the original review. This should ideally not be here and only on the pages that need these strings.

Hannah does have a [ticket](https://codedotorg.atlassian.net/browse/ACQ-2183) to move this.